en/docs/ipaas/micro-integrator/developing-integrations-with-integration-studio.md

```diff
@@ -0,0 +1,106 @@
+building and deploying integration artifacts in WSO2 Micro Integrator. It is an Eclipse-based IDE that provides a
+comprehensive set of tools for designing and testing integration flows and building integration artifacts. With WSO2
+Integration Studio, developers can create integration projects and develop integration artifacts in a visual,
+drag-and-drop manner, reducing the time and effort required to build and deploy integration flows. Additionally, the
+Studio provides features such as debugging, testing, and version control to help ensure the quality and reliability of
+integration flows.
+
+Creating an Integration project and Composite Exporter project in WSO2 Integration Studio allows developers to export
+their integration projects as a single deployment artifact, called a composite application. This composite application
+contains all the components and configurations required to deploy the integration project to WSO2 Micro
+Integrator. The Composite Exporter project helps to simplify the deployment process by allowing developers to package
+their integration projects as a single, self-contained artifact that can be easily deployed and managed. This can help
+improve the consistency and reliability of integration projects and simplify the overall deployment process.
+
+To get started with Integration Studio, see [WSO2 API Manager Documentation - Developing Integration Solutions](https://apim.docs.wso2.com/en/latest/integrate/develop/intro-integration-development/).
+
+## Integration Project Directory Structure
+
+Integration Project is a multi module maven project. It can contain multiple modules with WSO2 Synpase configurations
+and composite application module to export all the configurations as a single deployable artifact.
+Addition to the above OpenAPI definition file and libraries can be added optionally. OpenAPI definition can be added to
+the any location in the project and should be define it’s path when creating a component. Java libraries can be added to
+the directory created in the project root location with name libs.
+
+## OpenAPI support
+
+An [OpenAPI definition](https://spec.openapis.org/oas/v3.1.0#openapi-specification) can be developed to define your REST
+API. When a REST API is developed with integration studio and deployed on Choreo, it allows you to give the OpenAPI
+definition to attach to your Integration as an API component. Attaching an OpenAPI definition allows developers and
+consumers to understand the API.
+
+When a REST API is developed in WSO2 Integration Studio, developers can add an OpenAPI definition to the project. During
+component creation, the path to the OpenAPI Specification (OAS) file can be specified.
+
+![Open API File](/assets/img/ipaas/integration-studio/openapi_file.png){.cInlineImage-half}
+
+## Adding third party libraries
+
+Adding third-party jars to WSO2 Micro Integrator can bring several benefits to the development and deployment of
```

Comment:
Suggestion:
"Adding third-party jars to the Micro Integrator can provide the following benefits:

- Providing additional functionality that is not available in the core Micro Integrator distribution, allowing for greater flexibility and customization of integration solutions.
- Allowing the Micro Integrator to integrate with existing systems such as databases or other APIs, improving the overall integration capabilities.
- Providing optimized implementations of common functions, improving the performance of Micro Integrator.
- Developers can avoid having to re-implement commonly used functions, reducing development time and increasing efficiency."